src/sinks/s3_common/config.rs

```diff
@@ -299,7 +299,7 @@ pub fn build_healthcheck(bucket: String, client: S3Client) -> crate::Result<Heal
         match req {
             Ok(_) => Ok(()),
             Err(error) => Err(match error {
-                SdkError::ServiceError(inner) => match inner.raw().http().status() {
+                SdkError::ServiceError(inner) => match inner.into_raw().http().status() {
```

Comment:
**question**: I thought this failed to run due to ownership issues, which is why I ended up using `.raw()`. If that's not the case 🤷 maybe I was thinking of another place...

Reply:
Do you mean failed to compile?
Not sure... I didn't face problems with this when I compiled after resolving the merge conflicts.

Reply:
Word, it might have been a different place that this happened. IIRC there was an issue because `into_raw()` consumes Self.

If it's working here 👍